.github/workflows/lint.yaml

```diff
@@ -26,9 +27,24 @@ jobs:
         with:
           go-version: ${{ needs.tools.outputs.go-version }}
       - name: golangci-lint ${{ needs.tools.outputs.golangci-lint-version }}
-        uses: golangci/golangci-lint-action@v3.4.0
+        ##
+        # XXX: change this to the official action once multiple --out-format args are supported.
+        # See: https://github.com/golangci/golangci-lint-action/issues/612
+        ##
+        uses: smartcontractkit/golangci-lint-action@54ab6c5f11d66a92d14c3f7cc41ea13f676644bd # feature/multiple-output-formats-backup
+        with:
+          version: v1.52.1
```

Comment:
```suggestion
          version: v${{ needs.tools.outputs.golangci-lint-version }}
```

Reply:
There is a comment about why namely this version is being used, there is no way to provide the needed output format...